server/src/resolvers/usersResolver.ts

```diff
@@ -3,16 +3,20 @@ import * as ModuleRepo from "../repositories/Training/ModuleRepository";
 import { Privilege } from "../schemas/usersSchema";
 import { createLog } from "../repositories/AuditLogs/AuditLogRepository";
 import assert from "assert";
-import { createHash } from "crypto";
+import {createHash, randomBytes} from "crypto";
 import { ApolloContext } from "../context";
 import { UserRow } from "../db/tables";
 
 export function getUsersFullName(user: UserRow) {
   return `${user.firstName} ${user.lastName}`;
 }
 
-export function hashUniversityID(universityID: string) {
-  return createHash("sha256").update(universityID).digest("hex");
+export function hashUniversityID(universityID: string, salt: string) {
+  const hashed = createHash("sha256").update(universityID + salt).digest("hex");
+  return {
+    salt: salt,
+    hashedValue: hashed
+  };
```

Comment:
What's the benefit of returning the hash and salt separately, as an object? I think if we just returned `hashed + ":" + salt`, then we wouldn't have to stitch them together every time we want to check a UID.